static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -34,9 +34,16 @@ export type CtaUrls = {
     supportUrl?: string,
 };
 
-const membershipBaseURL = 'https://membership.theguardian.com/supporter';
-const contributionsBaseURL = 'https://contribute.theguardian.com';
+const useSupportDomain =
+    config.get('switches.ukSupporterTrafficToNewSupportFrontend') &&
+    geolocationGetSync() === 'GB';
 const supportBaseURL = 'https://support.theguardian.com/uk';
+const membershipBaseURL = useSupportDomain
+    ? supportBaseURL
+    : 'https://membership.theguardian.com/supporter';
+const contributionsBaseURL = useSupportDomain
```

Comment:
You'll need to make changes to the liveblog copy too:

https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-liveblog.js#L166

https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/acquisitions-copy.js#L44